tests/interchain/chainsuite/config.go

```diff
@@ -32,12 +32,14 @@ const (
 	CommitTimeout             = 2 * time.Second
 	TotalValidatorFunds       = 11_000_000_000
 	ValidatorFunds            = 30_000_000
-	ValidatorCount            = 1
-	FullNodeCount             = 0
-	ChainSpawnWait            = 155 * time.Second
-	CosmosChainType           = "cosmos"
-	GovModuleAddress          = "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn"
-	TestWalletsNumber         = 15 // Ensure that test accounts are used in a way that maintains the mutual independence of tests
+	// ValidatorCount is set to 2, so we have one active and one inactive (i.e., outside the active set) validator.
+	// Note that the provider has at most 1 validator (see `chain_spec_provider.go`).
+	ValidatorCount    = 2
```

Comment:
Is this done for al the tests or only for the new one introduced? We should avoid adding complexity to all tests. If you need multiple validators for a certain test, we should just add a new configuration.

Reply:
This will affect all tests in the suite that use this specification, i.e., the entire ProviderSuite. We could create a separate suite to support running tests on a suite with multiple nodes while keeping the existing suite where the provider chain has only one node (ProviderSuite, ProviderMultiNodeSuite).
Another option is to keep it as Karolos implemented and handle the optional nodes in the suite setup, so tests that need additional nodes can simply start the extra validator(s) as the interchaintest framework supports that.
The difference is that with separate suites, we don’t need to start/stop nodes dynamically, but the provider chain will be started twice. I'm not sure what the optimal solution would be.
If you'd like, we can leave it as Karolos implemented in this PR, and I can split it or add dynamic node management in a follow-up PR. I also plan to improve how "independent" test accounts are fetched, so I can handle everything in that PR.

Reply:
Sounds good. Let's do that. 
Regarding what option to choose, go with the one easier to implement.